Tests/ArcGISToolkitTests/PreplannedMapModelTests.swift

```diff
@@ -169,10 +169,14 @@ class PreplannedMapModelTests: XCTestCase {
         let areas = try await task.preplannedMapAreas
         let area = try XCTUnwrap(areas.first)
         let areaID = try XCTUnwrap(area.portalItem.id)
-        let mmpkDirectory = FileManager.default.preplannedDirectory(
-            forPortalItemID: portalItem.id!,
-            preplannedMapAreaID: areaID
-        )
+        let mmpkDirectory = URL.documentsDirectory
+            .appending(path: "OfflineMapAreas")
+            .appending(path: portalItem.id!.rawValue)
+            .appending(path: "Preplanned")
+            .appending(
+                path: areaID.rawValue,
+                directoryHint: .isDirectory
+            )
```

Comment:
Apple added a `URL` API specifically so that developers don't have to call `append` with each separate component:

```suggestion
            .appending(
                components: "OfflineMapAreas",
                "\(portalItem.id!)",
                "Preplanned",
                "\(areaID)/"
            )
```